### PR TITLE
Remove IE guard.

### PR DESCRIPTION
**Pull request by jdalton**
Related to the discussion over at https://github.com/jquery/sizzle/commit/28f4c67c508719cc76e2075dce0e4cbe0e325919#commitcomment-11778396.

### CODE REVIEW

**Comment by jdalton** [comment]
Whoever updates the CLA checks, I've signed the jq foundation cla before.

File: src/sizzle.js [hunk 1]
```diff
@@ -506,7 +506,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 	// Support: IE 9 - 11
 	// Accessing iframe documents after unload throws "permission denied" errors (jQuery #13936)
 	// Limit the fix to IE with document.documentMode and IE >=9 with document.defaultView
```
Comment (staabm) [inline src/sizzle.js]: Docs also need a change the

**Comment by mzgol** [comment]
> Whoever updates the CLA checks, I've signed the jq foundation cla before.

This is because the commit is signed as "jdalton". The name needs to match the one from the CLA signature.

File: src/sizzle.js [hunk 2]
```diff
@@ -505,8 +505,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 
 	// Support: IE 9 - 11
```
Comment (mzgol) [inline src/sizzle.js]: I guess this needs `, Edge BUILD_NUMBER` added.

**Comment by mzgol** [comment]
One comment; other than that & the commit signature :+1:.

**Comment by jdalton** [comment]
I've signed it with `jdalton` and `John-David Dalton` now.

File: src/sizzle.js (hunk 2 above)
Comment (jdalton) [inline src/sizzle.js]: I added `Edge` but left the build number off because it's not resolved, so open ended.

**Comment by mzgol** [comment]
I think it checks for a space inside a name so that all commits are signed with a full name but @scottgonzalez would know more.

File: src/sizzle.js [hunk 3]
```diff
@@ -503,10 +503,9 @@ setDocument = Sizzle.setDocument = function( node ) {
 	docElem = document.documentElement;
 	documentIsHTML = !isXML( document );
 
-	// Support: IE 9 - 11
+	// Support: IE 9 - Edge
```
Comment (mzgol) [inline src/sizzle.js]: This suggests Edge is another IE, I'd write:
```js
// Support: IE 9 - 11, Edge
```
We'll probably add some kind of versioning to Edge later (maybe by build numbers, see the discussion [from this comment forward](https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-117856894)) but until that's decided it doesn't matter too much.

**Comment by jdalton** [comment]
Ok, should be good to go.

**Pull request closed without merging**